strip out '-fp-model source' as a whole, rather than only stripping out '-fp-model' which leads to compilation errors:
"File not found: 'source'"

author: Kenneth Hoste (HPC-UGent), based on patch for Mesa 17.0.2 by Ward Poelmans (VUB)
--- configure.orig	2017-11-11 19:54:40.966320129 +0100
+++ configure	2017-11-11 19:59:25.921932866 +0100
@@ -22540,6 +22540,7 @@
         -e 's/[[:space:]]+-O[^[:space:]]*//g' \
         -e 's/[[:space:]]+-g[^[:space:]]*//g' \
         -e 's/-fno-rtti[[:space:]]/-Fno-rtti /g' \
+        -e 's/[[:space]]+-fp-model[[:space:]][^[:space:]]*//g' \
         -e 's/[[:space:]]+-f[^[:space:]]*//g' \
         -e 's/-Fno-rtti[[:space:]]/-fno-rtti /g' \
         -e 's/^[[:space:]]//' \
